documentation/features/context-menu.mdx

```diff
@@ -0,0 +1,48 @@
+---
+sidebarTitle: Context Menu
+title: Context Menu
+description:
+  Use context-aware menus to assist with your actions and improve your workflow.
+---
+
+Context-aware menus enable quick actions like creating variables or adding query
+parameters by selecting text or elements. The context menu offers three options:
+**Set as Variable**, **Add to Parameters**, and **Open Request in New Tab**.
+
+<Info>
+  The "Add to Parameters" option is available only when using query parameters.
+</Info>
+
+## Set as Variable
```

Comment:
Hey, thanks for making the changes. Could you please remove the quotes (>) from this section? They were only for your reference.